scripts/background.js

```diff
@@ -13,31 +13,31 @@ var excluded_urls = [
   "0.0.0.0",
   "127.0.0.1"
 ];
-var previous_RTurl="";
+var previous_RTurl = "";
 //Window ID of the Context-Windows
-var windowId1 =0;
-var windowId2 =0;
-var windowId3 =0;
-var windowId4 =0;
-var windowId5=0;
-var windowId6=0;
-var windowId7=0;
-var windowId8=0;
-var windowId9=0;
-var windowId10=0;
-var windowIdtest=0;
-var tabId1=0;
+var windowId1 = 0;
+var windowId2 = 0;
+var windowId3 = 0;
+var windowId4 = 0;
+var windowId5 = 0;
+var windowId6 = 0;
+var windowId7 = 0;
+var windowId8 = 0;
+var windowId9 = 0;
+var windowId10 = 0;
+var windowIdtest = 0;
+var tabId1 = 0;
 //Tab Id of the Context-tabs
-var tabId2=0;
-var tabId3=0;
-var tabId4=0;
-var tabId5=0;
-var tabId6=0;
-var tabId7=0;
-var tabId8=0;
-var tabId9=0;
-var tabId10=0;
-var windowIdSingle=0;
+var tabId2 = 0;
+var tabId3 = 0;
+var tabId4 = 0;
+var tabId5 = 0;
+var tabId6 = 0;
+var tabId7 = 0;
+var tabId8 = 0;
+var tabId9 = 0;
+var tabId10 = 0;
```

Comment:
We used to have 10 context windows but now we've got just 7.
The extra `tabId` and `windowId` vars need to be removed from everywhere (they are also present in some `if` conditions all over `background.js`.